backend/src/server/routes/v1/identity-router.ts

```diff
@@ -478,4 +478,27 @@ export const registerIdentityRouter = async (server: FastifyZodProvider) => {
       return { identityMemberships };
     }
   });
+
+  server.route({
+    method: "GET",
+    url: "/details",
+    config: {
+      rateLimit: readLimit
+    },
+    schema: {
+      response: {
+        200: z.object({
+          id: z.string(),
+          name: z.string(),
+          slug: z.string(),
+          role: z.string()
+        })
```

Comment:
these should be behind an "organization" property right? just in case we add more in the future,